wrapper-generator/src/test/kotlin/it/krzeminski/githubactions/wrappergenerator/versions/SuggestVersionsKtTest.kt

```diff
@@ -0,0 +1,98 @@
+package it.krzeminski.githubactions.wrappergenerator.versions
+
+import io.kotest.assertions.assertSoftly
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.data.Table2
+import io.kotest.data.forAll
+import io.kotest.data.headers
+import io.kotest.data.row
+import io.kotest.data.table
+import io.kotest.inspectors.forAll
+import io.kotest.matchers.shouldBe
+
+class SuggestVersionsKtTest : FunSpec({
+    test("Compare versions") {
+        val sortedVersions = listOf(
+            "v1.0.0",
+            "v1.0.9",
+            "v1.0.12",
+            "v1.2.0",
+            "v1.9.0",
+            "v1.13.1",
+            "v9.0.0",
+            "v12.0.0",
+        )
+        sortedVersions.sortedBy { VersionComparable(it) } shouldBe sortedVersions
+    }
+
+    context("Wrappers using major versions") {
+
+        test("No available versions") {
+            val testCases: Table2<String, String> = table(
+                headers("currentVersions", "availableVersions"),
+                row("v2", ""),
+                row("v2", "v1.1.1"),
+                row("v2", "v1.0.0, v1.1.1, v2.0.1, v2.1.0"),
+                row("v2", "v2.0.1"),
+                row("v2, v3", "v1.1.1, v2.0.1, v3.0.0, v3.4.0"),
+                row("v3, v2", "v1.1.1, v2.0.1, v3.0.0, v3.4.0"),
+                row("v12", "v9.0.1"),
+            )
+            testCases.forAll { current, available ->
+                suggestNewerVersion(current.split(", "), available.split(", ")) shouldBe null
+            }
+        }
+
+        test("New major version") {
+            assertSoftly {
+
+                suggestNewerVersion(
+                    listOf("v1"), listOf("v1.1.1", "v2.0.1", "v3.1.1")
```

Comment:
If these tests are for major versions, without minor and patch, I think it should say e.g.
```suggestion
                    listOf("v1"), listOf("v1", "v2", "v3")
```
Same below.

Reply:
That's not how it works.
The list `listOf("v1")` comes from the versions defined in `wrappersToGenerate`.
The list `listOf("v1.1.1", "v2.0.1", "v3.1.1")` is what comes from the HTTP call, and from that we have to understand that `v2` and `v3`  are available.

Reply:
I was wrong, logic updated, see comment below